src/main/kotlin/no/nav/familie/ef/sak/vilkår/VurderingController.kt

```diff
@@ -102,4 +104,29 @@ class VurderingController(
         gjenbrukVilkårService.gjenbrukInngangsvilkårVurderinger(request.behandlingId, request.kopierBehandlingId)
         return Ressurs.success(vurderingService.hentEllerOpprettVurderinger(request.behandlingId))
     }
+
+    @PostMapping("gjenbruk-enkelt-vilkar")
+    fun gjenbrukEnkeltVilkår(
+        @RequestBody request: EnkeltVilkårForGjenbrukRequest,
+    ): Ressurs<VilkårsvurderingDto> {
+        val behandlingForGjenbruk = gjenbrukVilkårService.finnBehandlingerForGjenbruk(request.behandlingId).first()
+
+        tilgangService.validerTilgangTilBehandling(behandlingForGjenbruk.id, AuditLoggerEvent.ACCESS)
+        tilgangService.validerTilgangTilBehandling(request.behandlingId, AuditLoggerEvent.UPDATE)
+        tilgangService.validerHarSaksbehandlerrolle()
+
+        val vilkårsVurderingForGjenbruk = gjenbrukVilkårService.gjenbrukInngangsvilkårVurdering(request.behandlingId, behandlingForGjenbruk.id, request.vilkårId)
+        return Ressurs.success(vilkårsVurderingForGjenbruk.tilDto())
+    }
+
+    @GetMapping("{behandlingId}/gjenbrukbare-vilkar")
+    fun hentAlleGjenbrukbareVilkårsvurderinger(
```

Comment:
```suggestion
    fun hentGjenbrukbareVilkårsvurderinger(
```
Småprik men blir det redundant med "Alle"? Vanlig praksis er jo at man alltid henter alle forekomster